src/components/templates/Playa/Playa.tsx

```diff
@@ -644,7 +644,7 @@ const Playa = () => {
                 <span className="img-vcenter-helper" />
                 <img
                   className="venue-icon"
-                  src={v.mapIconImageUrl || DEFAULT_MAP_ICON_URL}
+                  src={DEFAULT_MAP_ICON_URL}
                   alt={`${v.name} Icon`}
                 />
```

Comment:
Instead of just removing `v.mapIconImageUrl` here, I think we should either be changing it to one of the remaining non-legacy venue images, or removing the entire `img` tag here.

Since Playa is legacy code, perhaps we can just comment out this code instead. In doing that, it's probably worth adding back the `v.mapIconImageUrl ||` before commenting out the whole `img` tag, and then adding a comment explaining why it's commented out.

---

Is `DEFAULT_MAP_ICON_URL` also related to this same concept? Based on the usage here, it seems so. If so, we should probably be removing it from the codebase in this PR as well.

Reply:
Should we remove it here as well? https://github.com/sparkletown/sparkle/blob/staging/src/pages/Account/Venue/VenueMapEdition/Container.tsx#L238
https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/Playa/PlayaVenue.tsx#L42